spec.html

```diff
@@ -20501,7 +20507,8 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. If |LeftHandSideExpression| is neither an |ObjectLiteral| nor an |ArrayLiteral|, then
           1. Let _lref_ be ? Evaluation of |LeftHandSideExpression|.
           1. If IsAnonymousFunctionDefinition(|AssignmentExpression|) and IsIdentifierRef of |LeftHandSideExpression| are both *true*, then
```

Comment:
I know this doesn't have to be part of this PR, but we should update this line to match the other cases below.

```suggestion
          1. If IsAnonymousFunctionDefinition(|AssignmentExpression|) is *true* and IsIdentifierRef of |LeftHandSideExpression| is *true*, then
```

Reply:
I've gone ahead and opened a separate PR for it here: https://github.com/tc39/ecma262/pull/3330